From 1b065862f452d8c3be7e92ddb5ed4f0ef6dd0e88 Mon Sep 17 00:00:00 2001
From: Dong Aisheng <b29396@freescale.com>
Date: Thu, 9 Apr 2015 16:19:35 +0800
Subject: [PATCH 0952/1594] MLK-10629-3 dts: imx7d-sdb: add wifi-host property
 for usdhc2

commit 24341bbeaf54d69ca89de224c7bb75f45d5e7df9 from
git://git.freescale.com/imx/linux-2.6-imx.git

There's a Broadcom WiFi on usdhc2.
Add this property due to Broadcom WiFi driver needs call wifi_card_detect
function.

Signed-off-by: Dong Aisheng <b29396@freescale.com>
---
 arch/arm/boot/dts/imx7d-sdb.dts |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index 1236bcb..3359b20 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -899,6 +899,7 @@
 	keep-power-in-suspend;
 	tuning-step = <2>;
 	non-removable;
+	wifi-host;
 	status = "okay";
 };
 
-- 
1.7.5.4

